gulpfile.js

```diff
@@ -454,11 +454,11 @@ gulp.task('release', ['apps', 'clean-release'], function () {
     }
 
     if (platforms.indexOf('win32') !== -1) {
-        releaseWin('win32');
+        release_win('win32');
     }
     
     if (platforms.indexOf('win64') !== -1) {
-        releaseWin('win64');
+        release_win('win64');
```

Comment:
Any reason to move to snake_case here? The rest of the function names in this file (and most of the configurator) is camelCase.